numpy/_core/tests/test_multiarray.py

```diff
@@ -8290,7 +8297,8 @@ def test_padded_struct_array(self):
     def test_relaxed_strides(self, c=np.ones((1, 10, 10), dtype='i8')):
         # Note: c defined as parameter so that it is persistent and leak
         # checks will notice gh-16934 (buffer info cache leak).
-        c.strides = (-1, 80, 8)  # strides need to be fixed at export
+        with pytest.warns(DeprecationWarning):
+            c.strides = (-1, 80, 8)  # strides need to be fixed at export
```

Comment:
The ones we shouldn't just delete when finalizing would be nice to just replace immediately, but also not a big thing.

Reply:
Yes, the comment makes sense if a bit hard to follow.  That just means you need to move the `as_strided` call into the argument, though.